src/components/SearchBox/SearchBox.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
```

Comment:
Nit: I think it's more appropriate to call the comp `SearchField` than `SearchBox`

Reply:
pass